packages/smooth_app/lib/cards/data_cards/score_attribute_card.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:openfoodfacts/model/Attribute.dart';
+import 'package:smooth_app/cards/data_cards/attribute_chip.dart';
+import 'package:smooth_app/helpers/attributes_card_helper.dart';
+import 'package:smooth_app/themes/smooth_theme.dart';
+
+class ScoreAttributeCard extends StatelessWidget {
+  const ScoreAttributeCard({
+    required this.attribute,
+    required this.iconHeight,
+    this.barcode,
```

Comment:
I don't see the point of `barcode` here. Beyond the fact that it's not used in the code, I think it would look nicer if we say something like: "this is a score attribute card, regardless of the barcode".

Reply:
Ah yes, you are right, removed barcode.